spectral/README.md

```diff
@@ -9,26 +9,27 @@ A custom ruleset for [Spectral](https://stoplight.io/open-source/spectral/) foll
 To use this Spectral ruleset add the following to your `.spectral.yaml`:
 
 ```yaml
-extends: '@rhoas/spectral-ruleset'
+extends: 'node_modules/@rhoas/spectral-ruleset/dist/ruleset.yaml'
```

Comment:
That works under some very specific circumstances (definietly would not work with npx as I see now)

Reply:
ok, you're actually right. it worked initially for me but npx did ask for an install. which I did. I'll remove it